build/ninja-rules-cpp.sh

```diff
@@ -330,11 +330,22 @@ compile_one() {
         # Must have external 'time', and it must have -f
         # Notes: OS X has no time -f
         #        busybox time supports -f but not --format.
-        if command time -f '%e %M' true >/dev/null; then
-          set -- \
-            time -f "$out { elapsed: %e, max_RSS: %M }" -- \
-            "$@"
-        fi
+        case ${BASH_VERSION:-} in
+          3*)
+            # bash 3.2.x on macos has a bizarre bug that leads to weird
```

Comment:
I think it might be an interaction between `&` and `command time -f` ?

Or maybe it's an interaction between 3 things - `errexit`, `&`, and `command`

Reply:
Updated the comment to be a bit more specific. I left out the details about `&` because that is part of the script that sources this one. Doesn't hurt to be more explicit, though.